src/gemini_client.sh

```
@@ -0,0 +1,365 @@
+    
+    # Gemini CLIを実行（このCLIは標準入力とプロンプトオプションをサポート）
+    local result=""
+    local exit_code=0
+    
+    # --promptオプションを使用（Docker環境での互換性のため）
+    log_debug "--promptオプションでGemini CLI実行中..."
+    
+    # Gemini CLIを実行（ログ出力を完全に分離）
+    # Docker環境では--promptオプションを使用（stdin使用でハングするため）
+    local temp_output=$(mktemp)
+    if [[ -n "$TIMEOUT_CMD" ]]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)